rig/machine_control/machine_controller.py

```diff
@@ -2433,6 +2446,16 @@ def close(self):
             self.flush()
             self.closed = True
 
+    def free(self):
+        """Free the memory represented by the file-like."""
```

Comment:
That's an excellent point, I'll remove the commit for this from the PR for the time being.